web/js/map-OpenLayers.js

```diff
@@ -710,48 +697,29 @@ OpenLayers.Control.Click = OpenLayers.Class(OpenLayers.Control, {
         // Store pin location in form fields, and check coverage of point
         fixmystreet_update_pin(lonlat);
 
-        // Already did this first time map was clicked, so no need to do it again.
-        if (fixmystreet.page == 'new') {
-            return;
```

Comment:
Was this removed so that the map recentres upon a reclick?
If so, then the pan should be moved higher up instead; this needs to be present because e.g. it breaks the mobile reporting if you reclick on the map, as multiple OK/Try again buttons are created.